python/lsst/meas/mosaic/mosaicTask.py

```diff
@@ -910,14 +910,14 @@ def mosaic(self, dataRefList, tractInfo, ct=None, debug=False, diagDir=".",
 
         if self.config.doSolveWcs:
             if internal:
-                coeffSet = measMosaic.solveMosaic_CCD(order, nmatch, nsource,
+                coeffSet, matchVec, sourceVec, wcsDic, ccdSet = measMosaic.solveMosaic_CCD(order, nmatch, nsource,
```

Comment:
The returned values need to be assigned to the mosaicTask after this.  Currently all of them are set before.